packages/rspack/src/MultiCompiler.ts

```diff
@@ -550,8 +544,9 @@ export class MultiCompiler {
 			(compiler, cb) => {
 				compiler.close(cb);
 			},
-			// @ts-expect-error
-			callback
+			// cannot be resolved without any
+			// Type 'Error | null | undefined' is not assignable to type 'Error | null'
+			callback as any
```

Comment:
Can we reduce `as any` in this PR? I prefer using `as someType` instead of using `as any`